jobserver/jobs/hourly/dump_db.py

```diff
@@ -0,0 +1,37 @@
+import pathlib
+import subprocess
+import sys
+
+from django_extensions.management.jobs import HourlyJob
+from environs import Env
+
+
+env = Env()
+
+
+class Job(HourlyJob):
+    help = "Dump the database to storage for copying to local dev environments"  # noqa: A003
+
+    def execute(self):
+        database_url = env.str("DATABASE_URL")
+        output = pathlib.Path("/storage/jobserver.dump")
+
+        if not output.exists():
+            print(f"Unknown output path: {output}", file=sys.stderr)
+            sys.exit(1)
+
+        try:
+            with output.open() as f:
+                subprocess.check_call(
+                    [
+                        "pg_dump",
+                        "-Fc",
```

Comment:
Huh, this is not clear to me.

Is it "--format=c"? Or "-F" (default) and "-c" (clean)?

If we are using a custom format, maybe a) use `--format=c` and b) a comment as to why? I'm guessing because of the no acl/owner stuff?

Reply:
It's --format=c, will update with long form!

The custom format typically creates much smaller files.  The docs also note that it provides more flexibility when restoring.

Reply:
Do you think a comment would be useful here? I think I've only ever seen the custom format suggested when using pg_dump, so it's just the default style to me.

Reply:
So, I've generally just used text and gzipped it. So think a comment or maybe a link to doc explaining benefits would be good for future travellers